api/v1alpha1/meshsync_types_test.go

```diff
@@ -51,6 +52,21 @@ var _ = Describe("The test case for the meshsync CRDs", func() {
 					Name:      "default",
 				},
 			},
+			WatchList: corev1.ConfigMap{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: "v1apha1",
+					Kind:       "ConfigMap",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "watch-list",
+					Namespace: "default",
+				},
+				Data: map[string]string{
+					"blacklist": "blacklist",
+					"global":    "global-configs",
+					"local":     "global-configs",
```

Comment:
```suggestion
					"local":     "local-configs",
```

Reply:
@KiptoonKipkurui I think this is the reason why tests are failing.

Reply:
@MUzairS15 i have fixed the failing unit tests,made them more representative of how the configmap should look with blacklist and whitelisted resources